src/content/blog/2023-01-10-eslint-community-org.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: post
+title: Announcing ESLint's official Community organization
+teaser: We are happy to announce the official ESLint Community organization.
+tags:
+  - Community organization
+  - Updates
+authors:
+  - MichaelDeBoey
+categories:
+  - Announcements
+---
+
+In our
+["ESLint's 2022 year in review" blogpost](https://eslint.org/blog/2023/01/eslint-2022-year-review),
+[Nicholas](https://github.com/nzakas) already briefly mentioned it in the
+['Project kickoffs' section](https://eslint.org/blog/2023/01/eslint-2022-year-review#project-kickoffs),
+but I'm happy to now officially announce the all new official
+[ESLint Community GitHub organization](https://github.com/eslint-community).
```

Comment:
This will be easier to read with fewer links.

```suggestion
In our [2022 year in review](https://eslint.org/blog/2023/01/eslint-2022-year-review),
we briefly mentioned a couple of project kickoffs, and I'm excited now excited to announce the official
[ESLint Community GitHub organization](https://github.com/eslint-community).
```